From 89c5c339bd7ded8478a07fe83f563eb5f6f572f7 Mon Sep 17 00:00:00 2001
From: Ztj <ztj1993@gmail.com>
Date: Thu, 13 Aug 2020 11:57:39 +0800
Subject: [PATCH] disable update

---
 lib/core.ps1 | 1 +
 1 file changed, 1 insertion(+)

diff --git a/lib/core.ps1 b/lib/core.ps1
index a917f26..5b069aa 100644
--- a/lib/core.ps1
+++ b/lib/core.ps1
@@ -828,6 +828,7 @@ function last_scoop_update() {
 }
 
 function is_scoop_outdated() {
+    return $false
     $last_update = $(last_scoop_update)
     $now = [System.DateTime]::Now
     if($null -eq $last_update) {
-- 
2.26.2.windows.1

